nx.json

```diff
@@ -0,0 +1,81 @@
+{
+  "tasksRunnerOptions": {
+    "default": {
+      "runner": "@nrwl/nx-cloud",
+      "options": {
+        "cacheableOperations": [
+          "build",
+          "lint",
+          "typecheck",
+          "test-unit",
+          "test-build",
+          "test-docs",
+          "docs-build"
+        ],
+        "accessToken": "NjYwZGFlNWYtZGYyNS00MTIzLTkzYTEtYzAxZTZiYzUwYTVifHJlYWQtd3JpdGU="
+      }
+    }
+  },
+  "namedInputs": {
```

Comment:
See description for how `inputs` and `namedInputs` can help local dev and CI experience.